benchmark_runner/main/main.py

```diff
@@ -60,22 +59,26 @@
 
 
 @logger_time_stamp
-def azure_cluster_start_stop():
+def azure_cluster_operations():
```

Comment:
Pass `azure_cluster_operation` in as an argument rather than relying on its global value, which is harder to read.